samples/star-wars-api/HttpHandlers.fs

```diff
@@ -46,6 +47,7 @@ module HttpHandlers =
         task {
             let logger = ctx.RequestServices.CreateLogger moduleType
             let jsonSerializerOptions = ctx.RequestServices.GetRequiredService<IOptions<JsonOptions>>().Value.SerializerOptions
+            let serializeIt data = JsonSerializer.Serialize (data, jsonSerializerOptions)
```

Comment:
```suggestion
            let serializeWithOptions data = JsonSerializer.Serialize (data, jsonSerializerOptions)
```

Reply:
Changed in ea4faf5070cc800f132d3e2f8d4efa525530faaf .